docs/rst/user_manual/getting_started.rst

```diff
@@ -0,0 +1,102 @@
+.. include:: ../exports/alias.include
+
+.. _getting_started_tutorial:
+
+###############
+Getting Started
+###############
+
+|ddsrouter| is a cross-platform non-graphic application developed by eProsima and powered by Fast DDS
+that allow to create a communication bridge that connects two DDS networks that otherwise would be isolated.
+The main use case of the |ddsrouter| is to communicate two DDS networks that are physically or virtually separated
+and belong to different LANs, allowing those entities to publish and subscribe to local and remote topics indistinctly.
```

Comment:
```suggestion
|ddsrouter| is a cross-platform non-graphical application developed by eProsima and powered by Fast DDS
that allows to create a communication bridge that connects two DDS networks that otherwise would be isolated.
The main use case of the |ddsrouter| is to communicate two DDS networks that are physically or virtually separated
and belong to different LANs, allowing the entities of each network to publish and subscribe to local and remote topics indistinctly.
```

Reply:
:+1: